lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_ConvertLocationAndScaleParametersToPercentiles.py

```diff
@@ -311,7 +319,7 @@ def test_negative_percentiles(self):
         plugin = Plugin()
         msg = "NaNs are present within the result for the"
         with self.assertRaisesRegex(ValueError, msg):
-            plugin._mean_and_variance_to_percentiles(
+            plugin._location_and_scale_parameters_to_percentiles(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _location_and_scale_parameters_to_percentiles of a client class](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4576393)